lib/sales_analyst.rb

```diff
@@ -14,6 +15,14 @@ def average_items_per_merchant
     (@items.all.count / @merchants.all.count.to_f).round(2)
   end
 
+  def average_item_price_for_merchant(id)
```

Comment:
I think if we have time we could refactor this method a little bit but glad it all works and the tests pass